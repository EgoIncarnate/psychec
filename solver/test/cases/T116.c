void f() {
    char *p;
    if (p) {}
    T a = p;

    char *p2;
    if (!p2) {}
    T2 b = !p2;
}
