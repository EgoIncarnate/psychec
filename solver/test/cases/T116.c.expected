typedef  int T2 ;
typedef  char* T ;
